Cargo.toml

```diff
@@ -7,8 +7,8 @@ authors = [
 edition = "2018"
 
 [dependencies]
-kelvin = "0.18"
-nstack = "0.4"
+kelvin = "0.19.0"
```

Comment:
Is there any reason why you added the `.0` at the end again?

Reply:
No. Indeed it does nothing. Cargo will use the latest patch anyway..